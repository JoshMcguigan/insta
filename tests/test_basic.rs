extern crate insta;

use insta::{
    assert_debug_snapshot_matches, assert_json_snapshot_matches, assert_yaml_snapshot_matches,
};

#[test]
fn test_debug_vector() {
    assert_debug_snapshot_matches!("debug_vector", vec![1, 2, 3]);
}

#[test]
fn test_yaml_vector() {
    assert_yaml_snapshot_matches!("yaml_vector", vec![1, 2, 3]);
}

#[test]
fn test_json_vector() {
    assert_json_snapshot_matches!("json_vector", vec![1, 2, 3]);
}
