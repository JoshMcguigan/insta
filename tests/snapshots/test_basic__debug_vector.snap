---
created: "2019-03-03T12:32:23.110499Z"
creator: insta@0.6.3
source: tests/test_basic.rs
expression: "vec![1 , 2 , 3]"
---
[
    1,
    2,
    3
]
